mkt/developers/views.py

```diff
@@ -553,6 +553,7 @@ def ownership(request, addon_id, addon):
                     thread___addon=addon).delete()
 
         for author in user_form.deleted_objects:
+            author.delete()
```

Comment:
Oh this is the AddonUser model that's being deleted. That should be ok, I was worried about deleting something that cascaded with terrible consequences.